pkg/collector/reconcile/configmap.go

```diff
@@ -117,10 +117,19 @@ func expectedConfigMaps(ctx context.Context, params Params, expected []corev1.Co
 			updated.ObjectMeta.Labels[k] = v
 		}
 
-		if err := params.Client.Update(ctx, updated); err != nil {
+		// editting Update with Patch
```

Comment:
You can remove this comment, the code is simple enough to be understood without it.